pkg/gofr/datasource/sql/bind.go

```diff
@@ -0,0 +1,47 @@
+package sql
+
+import (
+	"fmt"
+	"strings"
+)
+
+const (
+	DialectMysql    = "mysql"
+	DialectPostgres = "postgres"
+)
+
+const (
+	UNKNOWN = iota
+	QUESTION
+	DOLLAR
+)
+
+func Rebind(dialect, query string) string {
+	switch bindType(dialect) {
+	case QUESTION, UNKNOWN:
+		return query
+	case DOLLAR:
+		queryFormat := strings.Replace(query, "?", "%v", -1)
+		count := strings.Count(query, "?")
+		replacement := make([]interface{}, count)
+
+		for i := 0; i < count; i++ {
+			replacement[i] = fmt.Sprintf("$%v", i+1)
+		}
+
+		return fmt.Sprintf(queryFormat, replacement...)
+	}
+
+	return query
```

Comment:
This would remain unused.

Can we keep `case QUESTION, UNKNOWN:` as the dafault case?

Reply:
Yes, but we have to suppress:
`pkg/gofr/datasource/sql/bind.go:23:2: missing cases in switch of type sql.BindVarType: sql.UNKNOWN, sql.QUESTION (exhaustive)`

Reply:
In that case an If statement can also work.

We can move to switch once we have any new bindVarType.